src/declarations/major/EnvVars.ts

```diff
@@ -3,6 +3,11 @@
 export default {
   nodeEnv: (process.env.NODE_ENV ?? ''),
   port: (process.env.PORT ?? 0),
```

Comment:
why do we have a duplicate declaration of PORT?

Reply:
Fixed.